src/requests/applicationPassword/delete.ts

```diff
@@ -0,0 +1,26 @@
+import { urlQueryBuilder } from "@netsu/js-utils";
+import { DefaultResponseModel } from "../../models";
+import { axiosConf, wdData } from "../../setup";
+import { URL } from "./config";
+import { DeleteAppPasswordModel } from "./models";
+
+/**
+ * Delete an Application Password
+ *
+ * https://docs.wildduck.email/api/#operation/deleteASP
+ *
+ * @param userId ID of the User
+ * @param asp ID of the Application Password
+ */
+export const deleteAppPassword = async (
+	userId: string,
+	aspId: string
+): Promise<DeleteAppPasswordModel> => {
+	const url = urlQueryBuilder(`${URL}/${aspId}`, {
```

Comment:
Please look at how variables placeholders inside the URL are handled in `src/requests/submission/post.ts`